redbot/cogs/general/general.py

```diff
@@ -224,49 +223,45 @@ def __init__(self):
             await ctx.send(_("I need the `Embed links` permission to send this."))
 
     @commands.command()
-    async def urban(self, ctx, *, search_terms: str, definition_number: int = 1):
-        """Urban Dictionary search
+    async def urban(self, ctx, *, word):
+        """Searches for urban dictionary entries using the unofficial urbandictionary api"""
```

Comment:
This help string is a bit long. Try to make the first line shorter than 72 characters, but the user should still know exactly what the command does from that line. Any further detail should be expanded after two line-breaks.